packages/main/src/Select.ts

```diff
@@ -518,30 +526,30 @@ class Select extends UI5Element {
 		}
 	}
 
-	_searchNextItemByText(text) {
+	_searchNextItemByText(text: string) {
 		let orderedOptions = this._filteredItems.slice(0);
 		const optionsAfterSelected = orderedOptions.splice(this._selectedIndex + 1, orderedOptions.length - this._selectedIndex);
 		const optionsBeforeSelected = orderedOptions.splice(0, orderedOptions.length - 1);
 
 		orderedOptions = optionsAfterSelected.concat(optionsBeforeSelected);
 
-		return orderedOptions.find(option => option.textContent.toLowerCase().startsWith(text));
+		return orderedOptions.find(option => option.textContent!.toLowerCase().startsWith(text));
```

Comment:
HTMLElement.textContent may be null by specification so i would prefer to make it ``` (option.textContent || "").toLowerCase...```